src/problem2.rb

```diff
@@ -0,0 +1,9 @@
+class Problem2
+  def identify_largest_numbers(array, no_elements_identify)
+    unless array.nil?
```

Comment:
Same as above see if you can drop the branching, branch prediction is a hard optimization don't make the processors live hard.